python/Ganga/GPIDev/Lib/Job/Job.py

```diff
@@ -52,6 +52,10 @@ def lazyLoadJobApplication(this_job):
 
 def lazyLoadJobObject(this_job, this_attr):
 
+    if stripProxy(this_job)._getRegistry():
+        if stripProxy(this_job)._getRegistry().has_loaded(stripProxy(this_job)):
+            return getattr(this_job, this_attr)
```

Comment:
This function should never be called from the proxy layer so the `stripProxy` should not be needed.